package.json

```diff
@@ -52,6 +52,7 @@
     "@webpack-cli/serve": "^1.6.0",
     "benchmark": "^2.1.4",
     "chai": "^4.2.0",
+    "editorconfig": "^0.15.3",
```

Comment:
```suggestion
```

[This npm package](https://github.com/editorconfig/editorconfig-core-js#readme) is not required. The editorconfig setting can be used directly in some IDEs and editors, or you can use it by installing an additional extension such as [vscode](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig).

See [No Plugin Necessary](https://editorconfig.org/#pre-installed) section and [Download a Plugin](https://editorconfig.org/#download) section.

Reply:
> The code formatter [prettier](https://prettier.io/) configuration used in this project is referring to the .editorconfig config.

> [This npm package](https://github.com/editorconfig/editorconfig-core-js#readme) is not required. The editorconfig setting can be used directly in some IDEs and editors, or you can use it by installing an additional extension such as [vscode](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig).  

Thank you for letting me know 😄 

## fix lt 
- Remove tabWidth option in prettierrc.js 
- Remove editorconfig module.